### PR TITLE
Enable auto-apply for run trigger runs

### PR DESCRIPTION
**Pull request by Uk1288**
### What
Update docs to indicate support of auto-apply for run triggers

### Screenshots
Screenshot 2023-06-22 at 4 35 55 PM

----------
Screenshot 2023-08-02 at 3 34 55 PM

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file. N/A
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.). N/A
- [x] Pages with related content are updated and link to this content when appropriate. N/A
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages. N/A
- [x] New pages have metadata (page name and description) at the top. N/A
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility. N/A
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars. N/A
- [x] UI elements (button names, page names, etc.) are bolded. N/A
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/api-docs/changelog.mdx [hunk 1]
```diff
@@ -9,9 +9,11 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-08-02
+* Updated [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) to support `auto-apply-run-trigger` attribute which controls whether run trigger runs should be auto applied.
```
Comment (uturunku1) [inline website/docs/cloud-docs/api-docs/changelog.mdx:13]:
```suggestion
* Update [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) to support `auto-apply-run-trigger` attribute which controls whether run trigger runs should be auto applied.
```

**Review by uturunku1 — CHANGES_REQUESTED** [review]
Screenshot 2023-08-09 at 3 04 01 PM
I was noticing the sentences for the changelog start in present tense. Aside from this change, I am ready to approve this PR!

File: website/docs/cloud-docs/workspaces/settings/run-triggers.mdx [hunk 2]
```diff
@@ -30,13 +30,13 @@ Creating run triggers requires admin access to the workspace. You must also have
 
 Under the "Source Workspaces" section, select the workspace you would like to connect as your source and click "Add workspace". You now have a run trigger established with your source workspace. Any run from that source workspace which applies successfully will now cause a new run to be queued in your workspace.
 
--> **Note:** Runs initiated in your workspace as the result of a run trigger connection to a source workspace will not auto-apply, regardless of your [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply) setting selection. You will need to manually apply these runs.
+~> **Important:** Runs initiated in your workspace as the result of a run trigger connection to a source workspace may auto-apply depending on your [auto-apply-run-trigger](/terraform/cloud-docs/workspaces/settings#run-triggers) setting selection.
```
Comment (brandonc) [inline website/docs/cloud-docs/workspaces/settings/run-triggers.mdx:33]: I found the message itself softened from "will never auto-apply" to "may auto-apply" but the severity went from "note" to "important"-- I think we should explain the interaction here but I also think there's nothing unexpected that needs a callout box.

```suggestion
## Run Triggers Auto-Apply Setting

Runs initiated by run trigger will not auto-apply unless the [auto-apply-run-trigger](/terraform/cloud-docs/workspaces/settings#run-triggers) setting is enabled. This setting operates independently of the workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply) setting.
```

File: website/docs/cloud-docs/api-docs/workspaces.mdx [hunk 3]
```diff
@@ -71,7 +71,8 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
...
+| `data.attributes.auto-apply`                  | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by VCS, UI or CLI, [with some exceptions](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                                                                      |
+| `data.attributes.auto-apply-run-trigger`      | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by run triggers. Not yet supported in Terraform Enterprise.                                                                                                                                                                                                                                                                                                   |
```
Comment (rkoron007) [inline website/docs/cloud-docs/api-docs/workspaces.mdx:75]: We only talk about present tense in docs, so we don't need the "yet":
```suggestion
| `data.attributes.auto-apply-run-trigger`      | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by run triggers. Not supported in Terraform Enterprise.                                                                                                                                                                                                                                                                                                   |
```

File: website/docs/cloud-docs/api-docs/workspaces.mdx [hunk 4]
```diff
@@ -677,7 +681,8 @@ Properties without a default value are required.
...
+| `data.attributes.auto-apply`                  | boolean        | (previous value) | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by VCS, UI or CLI, [with some exceptions](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                                                                      |
+| `data.attributes.auto-apply-run-trigger`      | boolean        | (previous value) | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by run triggers. Not yet supported in Terraform Enterprise.                                                                                                                                                                                                                                                                                                    |
```
Comment (rkoron007) [inline website/docs/cloud-docs/api-docs/workspaces.mdx:685]: Same as above:
```suggestion
| `data.attributes.auto-apply-run-trigger`      | boolean        | (previous value) | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by run triggers. Not supported in Terraform Enterprise.                                                                                                                                                                                                                                                                                                    |
```

File: website/docs/cloud-docs/workspaces/settings/index.mdx [hunk 5]
```diff
@@ -70,21 +70,29 @@ Changing your workspace's execution mode after a run has already been planned wi
...
+<a id="apply-method"></a>
+<a id="auto-apply-and-manual-apply"></a>
+
+### Auto-apply
 
 Whether or not Terraform Cloud should automatically apply a successful Terraform plan. If you choose manual apply, an operator must confirm a successful plan and choose to apply it.
 
+<!-- BEGIN: TFC:only name:auto-apply-run-trigger -->
+
+In addition to the main auto-apply setting (which affects runs created by the UI, API, CLI, and version control webhooks), Terraform Cloud also has a separate setting for runs created by [run triggers](/terraform/cloud-docs/workspaces/settings/run-triggers) from another workspace.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/settings/index.mdx:89]: I think this flows slightly better as two sentences:
```suggestion
The main auto-apply setting affects runs created by the Terraform Cloud user interface, API, CLI, and version control webhooks. Terraform Cloud also has a separate setting for runs created by [run triggers](/terraform/cloud-docs/workspaces/settings/run-triggers) from another workspace.
```

File: website/docs/cloud-docs/workspaces/settings/run-triggers.mdx [hunk 6]
```diff
@@ -30,13 +30,19 @@ Creating run triggers requires admin access to the workspace. You must also have
 
 Under the "Source Workspaces" section, select the workspace you would like to connect as your source and click "Add workspace". You now have a run trigger established with your source workspace. Any run from that source workspace which applies successfully will now cause a new run to be queued in your workspace.
 
--> **Note:** Runs initiated in your workspace as the result of a run trigger connection to a source workspace will not auto-apply, regardless of your [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply) setting selection. You will need to manually apply these runs.
+<!-- BEGIN: TFC:only name:auto-apply-run-trigger -->
+
+## Run Triggers Auto-Apply Setting
+
+Runs initiated by run trigger will not auto-apply unless the "Auto-apply run triggers" setting is enabled. This setting operates independently of the primary workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) setting.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/settings/run-triggers.mdx:37]: I'm assuming the UI actually says "Auto-apply run triggers"? If so then we can bold it like so:
```suggestion
Runs initiated by a run trigger do not auto-apply unless you enable the **Auto-apply run triggers** setting. This setting operates independently of the primary workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) setting.
```

**Review by rkoron007 — APPROVED** [review]
Small suggestions for style, but this looks great to me! As always, feel free to push back on anything that doesn't feel good to you ✨

**Pull request merged**